### DISCUSSION TITLE
Is this library a good fit to periodically get stock price for several internation stocks?

### DISCUSSION THREAD

**Discussion by gety9** [ranaroussi/yfinance#1867] [Q&A]
Title. International stocks (UK, Sweden, Hong Kong, etc)

I just need quote/price. Historical is fine (like 15-30 min delay). Very low call volume (like 10 stocks with 4 calls per day each).

If no, could you recommend better way?

**Comment by s-kust** [comment]
Take a look at alphavantage, it's also free
